lib/urn.js

```diff
@@ -31,7 +31,8 @@ module.exports = {
       'pbk:rsa:2048',
       'ble:1\.0',
       'nfc:1\.0',
-      'sn'
+      'sn',
+      'hash:argon'
```

Comment:
Thanks for the comment. Do we want the SDK to handle converting the input to hex?

Yes, the string (whether it's hex-encoded or not) will contain information about the Argon version.